setup.py

```diff
@@ -116,6 +116,7 @@
             'paasta_firewall_update=paasta_tools.firewall_update:main',
             'paasta_firewall_logging=paasta_tools.firewall_logging:main',
             'paasta_oom_finder=paasta_tools.oom_finder:main',
```

Comment:
Can you remove this script as this one replaces it? (I don't want to slightly different scripts for the same thing)